src/cpp.rs

```diff
@@ -639,7 +639,12 @@ pub mod root {
         #[repr(C)]
         #[derive(Debug, Copy, Clone)]
         pub struct ItemCommonParamFloat {
-            pub _address: u8,
+            pub _address: f32,
```

Comment:
For structs where the only field was previously `_address` and you're replacing it with a new field, change it to a [newtype](https://doc.rust-lang.org/stable/rust-by-example/generics/new_types.html). For example:

```rust
#[repr(transparent)]
#[derive(Debug, Copy, Clone)]
struct ItemCommonParamFloat(f32);
```

Note two things:

```rust
#[repr(transparent)]
```

as well as the use of a [tuple-styled struct](https://doc.rust-lang.org/book/ch05-01-defining-structs.html#using-tuple-structs-without-named-fields-to-create-different-types) with a single element.